packages/darma-vt/package.py

```diff
@@ -94,6 +94,10 @@ class DarmaVt(CMakePackage):
             description='Force use of OpenMP for threading')
     variant('use_std_thread', values=("0", "1"), default="0",
             description='Force use of std::thread for threading')
+    variant('build_tests', values=("0", "1"), default="0",
```

Comment:
Would it make sense to use spack boolean variants? e.g. https://spack.readthedocs.io/en/latest/packaging_guide.html#boolean-variants

That way users could use +build_tests and +build_examples which may be more intuitive

Reply:
Makes sense

Reply:
Done!